plugins/sqlalchemy/scripts/flytekit_install_dolt.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
should we get rid of this as well?

Reply:
yeah, sorry this is just a failure of detail-attentiveness on my part, the size of this PR discussion is getting out of hand. i'll make everything cleaner next time if you guys end up liking the dolt plugin PR

Reply:
oh no not at all, thank you for putting up with me!  Thanks for figuring out the sqlite test too.